Assets/Demo/DemoControls.cs

```diff
@@ -235,6 +668,7 @@ public InputControlScheme KeyboardMouseScheme
     public InputControlScheme GamepadScheme
     {
         get
+
```

Comment:
Generator seems to emit an extra newline here. Think we already have/had some whitespace issue but overall, would be great if the generator spits out code that results in zero formatting changes when running the Unity formatting tools.